packages/kbn-test/src/functional_tests/lib/run_ftr.js

```diff
@@ -0,0 +1,15 @@
+import { KIBANA_FTR_SCRIPT, PROJECT_ROOT } from './paths';
+
+export async function runFtr({ procs, configPath, cwd = PROJECT_ROOT }) {
+  const args = [KIBANA_FTR_SCRIPT, '--debug'];
```

Comment:
I think we should support the log-level flags in the CLI functions and pass the specific log level down to the functional test runner rather than hard coding `--debug`. I also think we should continue to support [the `--bail` option](https://github.com/elastic/kibana/blob/master/x-pack/dev-tools/functional_tests/lib/run_ftr.js#L20-L22) and possibly others, like `--grep`.

Reply:
What's the best way to go about doing this? To support [all the FTR cli options](https://github.com/elastic/kibana/blob/master/src/functional_test_runner/cli.js#L13-L20), they’d have to be basically treated as top level cli args for the scripts/functional_tests and scripts/functional_tests_server`? I think we are only supporting `--bail` like you mentioned (and I've added that back), but not others.

Would you agree with the follow-up task idea to pull FTR out of src and into a package (possibly kbn-test), and deal with it then?

Reply:
> To support [all the FTR cli options](https://github.com/elastic/kibana/blob/master/src/functional_test_runner/cli.js#L13-L20), they’d have to be basically treated as top level cli args for the scripts/functional_tests and `scripts/functional_tests_server`?

Yeah, I kind of assumed that over time this would replace the `functional_test_runner` cli anyway, and even in the short term will become the method people use to start the FTR, which is why I think it should support the flags that the FTR supports.